### PR TITLE
Added context menu sub items

### PR DESCRIPTION
**Pull request by LukeFlego**
Project sub items for starting a new entry with the chosen project

### CODE REVIEW

File: src/scripts/background.js [hunk 1]
```diff
@@ -1459,7 +1459,12 @@ var TogglButton = {
   },
 
   contextMenuClick: function (info, tab) {
-    TogglButton.createTimeEntry({"type": "timeEntry", "service": "contextMenu", "description": (info.selectionText || tab.title)}, null);
+    console.log(info);
```
Comment (imrel) [inline src/scripts/background.js]: is this logging necessary

File: src/scripts/background.js [hunk 2]
```diff
@@ -1623,7 +1628,31 @@ var TogglButton = {
...
+        "name": "No Client",
+        "projects": []
+      };
+      var projects = JSON.parse(Db.get("projects"));
+      for(var c in clients){
+        clients[c].projects = [];
+        for(var p in projects){
+          if(projects[p].cid == clients[c].id){
+            clients[c].projects.push(projects[p]);
+            console.log(projects[p]);
```
Comment (imrel) [inline src/scripts/background.js]: is this logging necessary

File: src/styles/settings.css [hunk 3]
```diff
@@ -6,7 +6,8 @@ body {
   display: none;
 }
 
-input[type="number"], input[type="time"]{
```
Comment (imrel) [inline src/styles/settings.css]: formatting and other layout changes should have been in separete PR instead to increase the possibility for them to accept the feature.

**Comment by tcrammond** [comment]
We're closing this PR for now to clean up the backlog; the issue #1006 remains open to be picked up at some point 🙇

**Pull request closed without merging**